Attempt to provide stream statistics as per #4934 request (#6558)
netdata/netdata: streaming/README.md

`/etc/netdata/edit-config stream.conf`). This file holds streaming configuration for both the
sending and the receiving Netdata.

You can add a [global] section on the master server's [stream.conf](stream.conf) and activate
additional slave statistics as follows.

```
[global]

> 				activate stream statistics = yes

Reviewer: Too much indentation here. Looks funky when rendered—Markdown respects all that whitespace.

```suggestion
    activate stream statistics = yes
```
```

```

> This will add a new streams section under netdata monitoring and provide five new charts with slave

Reviewer: netdata -> Netdata

> statistics

Reviewer: Put a colon `:` at the end here.

A "memory" chart that attempts to measure the amount of RAM allocated for the slaves in the master server (charts and dimensions)

A "streams" chart that reports the number currently incoming streams (slaves)

> A "processors" chart that reports the number of active processors of all slaves by incercepting the active processors HOST variable

Reviewer: ```suggestion
A "processors" chart that reports the number of active processors of all slaves by intercepting the active processors' `HOST` variable
```

A "metrics" chart to count all the dimensions provided by the slaves

> A "processes" chart for active processes reported by the slaves by intercepting the "system.active_processes" dimension

Reviewer: I think this might be better served as an unordered list.

> A "processes" chart for active processes reported by the slaves by intercepting the "system.active_processes" dimension

Reviewer: I think it makes sense to put dimensions in an inline code block.

```suggestion
A "processes" chart for active processes reported by the slaves by intercepting the `system.active_processes` dimension
```
